static-build/qemu-virtiofs/build-static-qemu-virtiofs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../../scripts/lib.sh"
+
+packaging_dir="${script_dir}/../.."
+qemu_virtiofs_repo="https://gitlab.com/virtio-fs/qemu.git"
+qemu_virtiofs_tag="virtio-fs-v0.3"
```

Comment:
eventually we should pull this from versions.yaml, right?

Reply:
@egernst yes, I need to put that tag on the versions.yaml of the runtime, I put a comment right now to clarify that

Reply:
@egernst , if you want I can create a PR at the `runtime` for the versions.yaml to put this tag